src/com/houarizegai/calculator/Calculator.java

```diff
@@ -496,7 +496,7 @@ private Calculator() {
                     } else {
                         inText.setText(String.valueOf(val));
                     }
-                    opt = '√';
+                    opt = "\u221A";
```

Comment:
`opt` variable is a char type, so you should put a signal quotes instead of double quotes.
will be: `opt = '\u221A';`